lib/jscsrc.json

```diff
@@ -17,7 +17,6 @@
     "switch"
   ],
   "requireDotNotation": true,
-  "requireLineBreakAfterVariableAssignment": true,
```

Comment:
This rule was broken in JSCS 1.12.0 (it was only checking for `var` statements), but was fixed in https://github.com/jscs-dev/node-jscs/pull/1191 (just not released yet).